config/routes.rb

```diff
@@ -3,7 +3,10 @@
                                        registrations: 'users/registrations',
                                        sessions: 'users/sessions'
                                      }
-  resources :articles
+  resources :articles do
+    resources :comments, only: [:create]
```

Comment:
I think it is better to allow every action here and remove [9 line](https://github.com/EvgeniiK/Mugenblog/pull/36/files#diff-21497849d8f00507c9c8dcaf6288b136R9). So your form wouldn't need condition anymore.